src/python/pants/core/goals/tailor_test.py

```diff
@@ -428,8 +428,12 @@ def test_tailor_rule_write_mode(rule_runner: RuleRunner) -> None:
             "conflict/BUILD": "fortran_library(sources=['f1.f90'])",
         }
     )
+    # Ensure the test invocation of Pants doesn't restart itself after it writes the tailored files,
+    # and then chokes on the invalid BUILD symbol.
     result = rule_runner.run_goal_rule(
-        TailorGoal, args=["--alias-mapping={'fortran_library': 'my_fortran_lib'}", "::"]
+        TailorGoal,
+        global_args=["--no-watch-filesystem", "--no-pantsd"],
+        args=["--tailor-alias-mapping={'fortran_library': 'my_fortran_lib'}", "::"],
     )
```

Comment:
This is not an integration test, so no external processes or `pantsd` are spawned: everything runs in process, and file invalidation happens synchronously during the `write_files` call. So these arguments have no effect.